### ISSUE TITLE
Highlighting incomplete

### ISSUE THREAD

**Issue by cseelus** [hhsnopek/vim-sugarss#4]
Hi @hhsnopek , thanks for you effort in creating aSugarSS syntax file for VIM .

This is how I expected it would highlight:

![screen shot 2017-12-09 at 19 20 31](https://user-images.githubusercontent.com/3188392/33798296-24d230b0-dd16-11e7-90ad-a69c5c40f295.png)
(Thats .sass, but it should look similar)

This is how it actually looks:

![screen shot 2017-12-09 at 19 20 01](https://user-images.githubusercontent.com/3188392/33798302-371c9544-dd16-11e7-961b-40aa46c74b09.png)

Inspecting the highlights with synstack shows that attributes have no highlight assigned at all (except for units), and the highlighting for IDs is disabled after the `_` sign.

Is there anything wrong on my side or is this just the current state of affairs for vim-sugarss? 🤔

**Comment by hhsnopek** [comment]
Ahh this is happening locally for me as well - My regex don't account all valid naming characters. I might be able to get this in later today

**Comment by cseelus** [comment]
I set up VIM to use highlighting for SASS for SSS files for now, which works quite well. 

Maybe (parts of) their syntax file could be used for vim-sugarss as well.

**Comment by hhsnopek** [comment]
yeah - anyone could easily use stylus or sass highlighting in place of vim-sugarss. I just wanted to pull out/use the bare minimum with sugarss

**Comment by toastal** [comment]
Similar happens with `-` separators. Also units inside other words like highlighting the `ex` in `display: flex`. (Tiny bug, but regex matches `ex` twice)
